composer.json

```diff
@@ -285,6 +285,9 @@
         ],
         "unit-tests": [
             "@php -d date.timezone=UTC ./vendor/phpunit/phpunit/phpunit -c tests/Unit/phpunit.xml"
+        ],
+        "phpstan": [
```

Comment:
cs fixer wants to move this before "restore-test-db" 😓

Reply:
Yup, closed laptop already. I'll fix it later :)